.travis.yml

```diff
@@ -1,17 +1,40 @@
 language: node_js
-
 node_js:
   - 8.9.4
 
+branches:
+  only:
+    - master
+    - /v\d+\.\d+\.d+$/
+
+before_install:
+  - npm install -g npm@latest
+install:
+  - npm ci
```

Comment:
I'm gonna revert this section; `npm ci` is only marginally faster (~75 seconds versus ~90), and the `before_install` step takes at least ~20 seconds so we don't gain anything. A regular `npm install` will be simpler.